modules/osdk-multi-arch-node-preference.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-multi-arch-support.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-multi-arch-node-preference_{context}"]
+= Using preferred node affintiy rules to configure support for multi-architecture compute machines for Operator project
+
+If your Operator performs better on particular architectures, you can configure preferred node affinity rules to schedule pods to nodes to the specified architectures.
+
+.Prerequisites
+
+* An Operator project created or maintained with Operator SDK {osdk_ver} or later.
+* You have built a manifest list defining the platforms your Operator supports.
+* You have set your Operator's required node affinity rules.
+
+.Procedure
+
+. Search your Operator project for Kubernetes manifests that define pod spec and pod template spec objects.
++
+.Example Kubernetes manifest
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: s1
+spec:
+  containers:
+    - name: <container_name>
+      image: docker.io/<org>/<image_name>
+----
+
+.  Set your Operator's preferred node affinity rules in the Kubernetes manifests that define pod spec and pod template spec objects, similar to the following example:
++
+.Example Kubernetes manifest
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: s1
+spec:
+  containers:
+    - name: <container_name>
+      image: docker.io/<org>/<image_name>
+  affinity:
+      nodeAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution: <1>
```

Comment:
Hi @michaelryanpeter that's ok to move forward for another jira, but what we say here seems still a bit unsorted to me, but maybe I miss some context.

Why are we talking more about preferred affinity and missing the required affinity which is the critical one?

@jaypoulz @Prashanth684 any context I' missing here?


https://github.com/openshift/openshift-docs/pull/64355/files#diff-8c533024c02a4b9ed134ba9f297cef3fe2ab12369d09fe1f6dd8ba4661e36155R43-R57

Reply:
Ignore the thread... I misread.